### ISSUE TITLE
[BUG] Importing @/tokens.stylex with stylex.create causes Panic error

### ISSUE THREAD

**Issue by bonttimo** [Dwlad90/stylex-swc-plugin#463]
### Describe the bug

When importing a **tokens.stylex.js** file into a **"client component"**, a panic error is intermittently thrown:` Error: Panic occurred during transformation: Cannot resolve file path: @/tokens.stylex`

The StyleX documentation says: **[3. No other exports are allowed in the file](https://stylexjs.com/docs/learn/theming/defining-variables/#rules-when-defining-variables)** So this is probably why the error occurs. Currently, the standard StyleX Next.js setup does not throw an error when creating **stylex.create**.

Here is a minimal reproduction of the error.

**layout.js**
```javascript
import "@/globals.css";
import * as stylex from "@stylexjs/stylex";

export default function RootLayout({ children }) {
    return (
        <html lang="en" {...stylex.props(styles.html)}>
            <body>{children}</body>
        </html>
    );
}

const styles = stylex.create({
    html: {
        display: "flex",
        flexDirection: "column",
        backgroundColor: "#f0f0f0",
    },
});
```

**page.js**
```javascript
"use client";

import * as stylex from "@stylexjs/stylex";
import { layout, spacingTokens } from "@/tokens.stylex";

export default function Home() {
    return (
        <div {...stylex.props(styles.page, layout.column, spacingTokens.spacing100)}>
            <p>Hello World</p>
        </div>
    );
}

const styles = stylex.create({
    page: {
        display: "flex",
        flexDirection: "column",
    },
});
```

**tokens.stylex.js**
```javascript
import * as stylex from "@stylexjs/stylex";

const spacing = {
    100: "0.25rem",
    200: "0.5rem",
    // ....
};

export const layout = stylex.create({
    column: { display: "flex", flexDirection: "column", width: "100%", height: "auto" },
    row: { display: "flex", flexDirection: "row", width: "100%", height: "auto" },
});

export const spacingTokens = stylex.create({
    spacing100: { padding: spacing[100] },
    spacing200: { padding: spacing[200] },
});
```
**next.config.mjs**
```javascript
import path from "path";
import stylexPlugin from "@stylexswc/nextjs-plugin";
import { fileURLToPath } from "url";

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);
const rootDir = __dirname;

export default stylexPlugin({
    rsOptions: {
        dev: process.env.NODE_ENV !== "production",
        aliases: {
            "@/*": [path.join(rootDir, "*")],
        },
        useRemForFontSize: true,
        unstable_moduleResolution: { type: "commonJS" },
        useCSSLayers: true,
    },
})({
    trailingSlash: true,
});
```

**postcss.config.mjs**
```javascript
import path from "path";
import { fileURLToPath } from "url";

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);
const projectRoot = __dirname;

const config = {
    plugins: {
        "@stylexswc/postcss-plugin": {
            include: ["app/**/*.{js,jsx,ts,tsx}", "components/**/*.{js,jsx,ts,tsx}"],
            useCSSLayers: true,
            rsOptions: {
                aliases: { "@/*": [path.join(projectRoot, "*")] },
                unstable_moduleResolution: { type: "commonJS" },
                dev: process.env.NODE_ENV === "development",
                genConditionalClasses: true,
                treeshakeCompensation: true,
            },
        },
        autoprefixer: {},
    },
};
export default config;
```

### Error stack

The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
 ⨯ ./src/app/page.js
Error: Panic occurred during transformation: Cannot resolve file path: @/tokens.stylex
Import trace for requested module:
./src/app/page.js
 ○ Compiling / ...
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
 ⨯ ./src/app/page.js
Error: Panic occurred during transformation: Cannot resolve file path: @/tokens.stylex
Import trace for requested module:
./src/app/page.js
The application panicked (crashed).
Message:  Cannot resolve file path: @/tokens.stylex
Location: crates/stylex-shared/src/shared/structures/state_manager.rs:895

Backtrace omitted.

Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
 GET /_next/static/webpack/8e032a0f3e769080.webpack.hot-update.json 500 in 2948ms
 ⚠ Fast Refresh had to perform a full reload due to a runtime error.
 GET / 500 in 219ms
 GET / 500 in 5ms
<w> [webpack.cache.PackFileCacheStrategy] Caching failed for pack: Error: ENOENT: no such file or directory, rename '/Users/USERNAME/Sites/localhost/stylex_bug/.next/cache/webpack/client-development-fallback/0.pack.gz_' -> '/Users/USERNAME/Sites/localhost/stylex_bug/.next/cache/webpack/client-development-fallback/0.pack.gz'
<w> [webpack.cache.PackFileCacheStrategy] Caching failed for pack: Error: ENOENT: no such file or directory, rename '/Users/USERNAME/Sites/localhost/stylex_bug/.next/cache/webpack/client-development-fallback/0.pack.gz_' -> '/Users/USERNAME/Sites/localhost/stylex_bug/.next/cache/webpack/client-development-fallback/0.pack.gz'

### OS

macOS  15.4.1 (24E263)

### Browser

Chrome 137.0.7151.69 (Official Build) (arm64)

### Compiler version

@stylexswc/rs-compiler 0.8.1

### Bundler plugin(s) version

@stylexswc/nextjs-plugin 0.8.1

### To Reproduce

1. Import a **tokens.stylex** file that includes a **stylex.create** definition into a **use client** component.

### Additional information

- [x] I have checked with the latest version

**Comment by bonttimo** [comment]
Somehow, I am still getting the same error. I am running **Node: v22.13.1**  and **NPM: 10.9.2**

I did some testing and noticed that 
```javascript
import { layout } from "../tokens.stylex"; // This works
import { layout } from "@/tokens.stylex"; // This does not
import Test from "@/Test"; // This works
```


Just for reference: **Test.js**
```javascript
const Test = () => <div>Test</div>;
export default Test;
```
**jsconfig.json**
```javascript
{
  "compilerOptions": {
    "paths": {
      "@/*": ["./src/*"]
    }
  }
}
```

Now I’m not entirely sure what's causing the error anymore 🙃

**Comment by Dwlad90** [comment]
> import { layout } from "@/tokens.stylex"; // This does not

If the breakdown occurs only with aliases, then the problem is where in the settings of their paths. Something like what I wrote above

Can you send a link to the project where the error is occurring? It would be easier and faster to figure out what exactly is happening.

**Comment by Dwlad90** [comment]
I replaced "app" with "src" in the alias paths (postcss + nextjs) and everything worked, here is a [link to the corrected sandbox](https://codesandbox.io/p/devbox/stylex-swc-forked-45ytc4)
